files/fielddata/wazaoshie/waza_oshie.json.txt

```diff
@@ -40,14 +40,14 @@
 
     .rodata
 
-    .global gWazaOshieMoves
-    .type gWazaOshieMoves,@object
-gWazaOshieMoves:
+    .global gMoveTutorMoves
+    .type gMoveTutorMoves,@object
+gMoveTutorMoves:
 ## for set in tutor
     oshie_begin
 ## for move in set.moves
     oshie_add {{ move }}
 ## endfor
     oshie_end
 ## endfor
-    .size gWazaOshieMoves,.-gWazaOshieMoves
+    .size gMoveTutorMoves,.-gMoveTutorMoves
```

Comment:
can we change this file's name too while we're at it?

Reply:
that might be a bit more difficult, since the stuff is passed into o2narc iirc